### PR TITLE
react function components

### PR DESCRIPTION
**Pull request by louiewhitz**
Screenshot 2022-10-26 at 2 45 32 PM

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+function CustomButton(props) {
+  // eslint-disable-next-line react/no-unescaped-entities
+  return <button>"Click Me!"</button>;
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: Inside of JSX you don't need quotes around text. Notice in your browser the button text has the quotes around the text, we don't want that.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

File: react-function-components/src/index.jsx (hunk 1 above)
Comment (louiewhitz) [reply react-function-components/src/index.jsx]: Yea, I was wondering about that - just saw in the instructions it had double quotes I will now remove. Thank you!

**Comment by louiewhitz** [comment]
Screenshot 2022-10-26 at 3 00 03 PM

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**